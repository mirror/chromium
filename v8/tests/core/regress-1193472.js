// Fails with an assertion failure in debug mode.

function f() {
  try {
    throw 42;
  } finally {
    return 87;
  }
}

f();

// Used to crash with an assertion failure because
// the externally caught flag was true as a left-over
// from calling f.
print(1234);

 
