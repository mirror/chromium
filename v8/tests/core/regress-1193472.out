regress-1193472.js:5: Uncaught 42
    throw 42;
    ^
1234
